devtools/platform-descriptor-json/src/main/resources/templates/README.maven.ftl

```diff
@@ -0,0 +1,29 @@
+# ${project_artifactId} project
+
+This project uses the Supersonic Subatomic Java Framework named Quarkus.
+
+If you want to learn more about this framework, please check this website: https://quarkus.io
+
+## Running the application in dev mode
+
+You can run your application in dev mode that will allow to have hot reload feature using:
+```
+./mvnw compile quarkus:dev
+```
+## Packaging and running the application
+
+The application is packageable using `./mvnw package`. It produces 2 jar files in `/target`:
+- `${project_artifactId}-${project_version}.jar` - containing just the classes and resources of the projects, it’s the regular artifact produced by the Maven build;
+- `${project_artifactId}-${project_version}-runner.jar` - being an executable _jar_. Be aware that it’s not an _über-jar_ as the dependencies are copied into the `target/lib` directory.
+
+The application is now runnable using `java -jar target/${project_artifactId}-${project_version}-runner.jar`.
+
+## Creating native executable
+
+You will be able to create a native executable using: `./mvnw package -Pnative`.
+
+Or you can use docker to build the native executable using: `./mvnw package -Pnative -Dnative-image.docker-build=true`.
```

Comment:
```suggestion
Or you can use docker to build the native executable using: `./mvnw package -Pnative -Dquarkus.native.container-build=true`.
```

Let's update the command native image build system props to match the new way - https://quarkus.io/guides/maven-tooling#build-a-container-friendly-executable